<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordLayout</class>
 <widget class="QWidget" name="PasswordLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Protection</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="buttonGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="noPasswordRadioButton">
        <property name="text">
         <string>&amp;No protection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="passwordRadioButton">
        <property name="text">
         <string>Protect basket with a &amp;password</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="publicPrivateRadioButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Protect basket with private &amp;key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="keyCombo"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
